src/common_modules/vsts-cli-build-common/vsts/cli/build/common/build.py

```diff
@@ -18,7 +18,7 @@
 logger = get_logger(__name__)
 
 def build_queue(definition_id=None, definition_name=None, branch=None, variables=None, open_browser=False,
-                team_instance=None, project=None, detect=None, source_branch=None):
+                team_instance=None, project=None, detect=None, source_branch=None, commit_id=None):
```

Comment:
Weren't these changes already merged previously?

Reply:
No these were not merged. You said earlier lets not merge and wait for other changes.

Reply:
I did. https://github.com/Microsoft/vsts-cli/pull/244
I am just curious why it didn't show you conflict.